src/main/java/com/libraryapp/Book.java

```diff
@@ -0,0 +1,35 @@
+package com.libraryapp;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book{
+    @Id
+    @SequenceGenerator(
+            name = "book_id_sequence",
+            sequenceName = "book_id_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "book_id_sequence"
+    )
+    private Integer bookId;
+    private String author;
+    private String title;
+    private Boolean availability;
+
+    public Book(Integer bookId,
```

Comment:
Lombokowa adnotacja `@Data` generuje też jakieś konstruktory, jeśli nie takie jak potrzebujesz to możesz użyć `@AllArgsConstructor` i `@NoArgsConstructor`